api/endpoints/taxitrips.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint, jsonify, request
+from api.metrics.taxitrips import TaxiTripMetrics
+
+#creates blueprint for endpoints related to taxi trips
+def make_blueprint(connection):
```

Comment:
Making a new blueprint wasn't necessary, what you added to `api/endpoints/community.py` was all that was needed to add those metrics to the API.

Sorry if I caused confusion by saying to look at rideshare as an example. I should have said the rideshare metric in the community and timeline endpoints, not the rideshare endpoint/blueprint.

For your other custom metrics for your question, we have a file called `api/endpoints/question.py` that you can use. Or since you made this new blueprint, we can use this, that works too!

There are some other changes to make here, like removing the support_metrics map in this file (it's not being used by the code) and registering this blueprint with the server (this happens in `api/server.py`, but those can be done later when you work on your question.